### DISCUSSION TITLE
Outdoor Camera Support

### DISCUSSION THREAD

**Discussion by ImTryingToLearnStuff** [mrlt8/docker-wyze-bridge#109] [Q&A]
This works great and I have waited a long time for something like this!  I even created a GIT account just so I can ask some question.  Are there any plans to support Wyze outdoor wire-free cameras (or maybe I am doing something incorrectly)?  

Currently I am getting the following errors `[Driveway] IOTC_ER_FAIL_SETUP_RELAY`

**Comment by mrlt8** [comment]
I believe the outdoor cameras are put into sleep mode and require an extra "wake" command before connecting to the camera. Unfortunately, I don't have an outdoor camera so can't test it out.

**Reply by ImTryingToLearnStuff** [reply]
Thanks for the fast reply, that is to bad since having the ability to see wyze outdoor cameras would make this solution even better.

Again, great work on this I am very glad I found it.  I have it running in a docker container for a few days now with no issues 🎉
